lib/board.rb

```diff
@@ -0,0 +1,68 @@
+require 'pry'
+class Board
+
+  attr_accessor :cells
+
+  def initialize
+    @cells = Array.new(9, " ")
+  end
+
+  def reset!
+    @cells = Array.new(9, " ")
+  end
+
+
+  def display
+      puts " #{@cells [0]} | #{@cells [1]} | #{@cells [2]} "
+      puts "-----------"
+      puts " #{@cells [3]} | #{@cells [4]} | #{@cells [5]} "
+      puts "-----------"
+      puts " #{@cells [6]} | #{@cells [7]} | #{@cells [8]} "
+  end
+
+  def position(input)
+    board_position = input.to_i - 1
+    @cells[board_position]
+  end
+
+  def full?
+    @cells.none? do |check|
+      if check == " "
+        true
```

Comment:
Avoid returning boolean literals from `if` statements, relying instead on the implicit return value of the boolean expression:

`cells.none? {|cell| cell == " "}`